src/libraries/Common/src/System/Net/Http/aspnetcore/CopyToAspNetCore.cmd

```diff
@@ -10,5 +10,6 @@ IF [%remote_repo%] == [] (
 
 echo ASPNETCORE_REPO: %remote_repo%
 
-robocopy . %remote_repo%\src\Shared\runtime /MIR
-robocopy .\..\..\..\..\..\tests\Tests\System\Net\aspnetcore\ %remote_repo%\src\Shared\test\Shared.Tests\runtime /MIR
+REM https://superuser.com/questions/280425/getting-robocopy-to-return-a-proper-exit-code
+(robocopy . %remote_repo%\src\Shared\runtime /MIR) ^& IF %ERRORLEVEL% LSS 8 SET ERRORLEVEL = 0
```

Comment:
The superuser link indicates that a 0x04 exit code is still some sort of error.

> 0×04 Some Mismatched files or directories were detected. Examine the output log. Housekeeping is probably necessary.

I'm not sure what "Some Mismatched files or directories were detected" actually means. Is this something we can safely ignore?

Reply:
Should be fine, we'll pick it up in the diff step.